lib/environment_helpers/enumerable_helpers.rb

```diff
@@ -0,0 +1,31 @@
+module EnvironmentHelpers
+  module EnumerableHelpers
+    VALID_TYPES = %i[strings symbols integers file_paths]
```

Comment:
[suggestion] Can we use `pathnames` here instead? I think that's the only way people will understand what they'll actually get when they use that type (or we could drop it, it seems less _basic_ than the other types we handle elsewhere. That would also turn that hash of lambdas into a hash of symbols to `send`).

Reply:
It is definitely less basic, but not difficult to implement, so I threw it in. It seems reasonable to just _not_ include it for now.